syntax = "proto3";

import "service/DataService.proto";
package apb.api.companion;

option java_package = "jp.co.apb.server.api.proto.companion";
option java_multiple_files = true;
option csharp_namespace = "Art.Framework.ApiNetwork.Grpc.Api.Companion";

message EnhanceRequest {
  string user_companion_uuid = 1;
  int32 add_level_count = 2;
}

message EnhanceResponse {
  map<string, .apb.api.data.DiffData> diff_user_data = 99;
}

service CompanionService {
  rpc Enhance(.apb.api.companion.EnhanceRequest) returns (.apb.api.companion.EnhanceResponse);
}

